citi-onboarding/acervusej-onepage
# Update/contact us button

**Things I've done:**

- Change the style of the button

**How to test it:**

- First download the node modules on the client and the server. If you have already done that, then:
- Change the values on .env.
- Go to client on the terminal (cd client) and put npm run start.
- On the browser write _http://localhost:3000/_.
- See if the button is the same as the mockup.
- Look at the code to see if it has everything that it should have.
